package.json

```diff
@@ -40,6 +40,9 @@
     "test": "nyc mocha --recursive --exit",
     "posttest": "eslint .",
     "report-coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
-    "beautify": "eslint . --fix"
+    "beautify": "eslint . --fix",
+    "mkdirs": "mkdir -p vendor/aws/",
```

Comment:
This might not work in Windows. Maybe better to have the `node-builder.js` script create the directory if it does not exist yet, using `fs` and `path` so that it works cross platform.

Regarding the path for the generated output: is `vendor` a common one? Looking at the [node module path conventions](https://gist.github.com/tracker1/59f2c13044315f88bee9) that @DrGirlfriend recently shared the most closest would be `build` (but not checked in).

`src` fits only half-way - as we don't want to check it in (I think it's meant for stuff like coffeescript that needs to be compiled first).

Anyway, what I want to say is we control the folder, so we can make it e.g. shorter so that it is easier to create on all OS.

Reply:
I'd say build should contain the build script - but currently we may have name collisions (since we used build previously). I used vendor to contain the compiled aws-sdk version, since it comes from aws. I followed this logic (details: https://tommcfarlin.com/the-vendor-directory/).

Reply:
Yeah, `build` is probably not ideal. (I believe it is only `.gitignored` because we copy/paste that file around and our workers need it since `nui` and `aio asset-compute` treat it as temporary build folder. But still, probably best to avoid such mixed usage across our repos.)

FWIW, that link isn't really about node modules... (it's about a folder for the client-side resources in their php based website).

Reply:
Still, the explication and name justification make sense!